python-clusters/create-eks-cluster/cluster.py

```diff
@@ -59,28 +59,51 @@ def start(self):
             args = args + get_region_arg(connection_info)
             args = args + ['--full-ecr-access']
 
-            subnets = networking_settings.get('subnets', [])
+            subnets = map(networking_settings.get('subnets', []), lambda subnet_id: subnet_id.strip())
```

Comment:
in `map` method, first argument should be the lambda expression and second the list on which we apply it